go/prometheusexporter/collector/collector.go

```diff
@@ -1134,7 +1134,7 @@ func extractExceptions(request map[string]interface{}) []string {
 
 	switch el := exceptions.(type) {
 	case []interface{}:
-		exList := make([]string, len(el))
+		exList := make([]string, 0, len(el))
```

Comment:
this still doesn't take care of empty strings.

Reply:
I added an additional check to check for empty string elements